numalogic/udfs/postprocess.py

```diff
@@ -122,7 +122,11 @@ def exec(self, keys: list[str], datum: Datum) -> Messages:
             load_latest=LOAD_LATEST,
             vertex=self._vtx,
         )
-        postproc_tx = self.postproc_factory.get_instance(postprocess_cfg)
+        postproc_tx = (
+            self.postproc_factory.get_instance(postprocess_cfg) if postprocess_cfg else None
+        )
+        if not postproc_tx:
```

Comment:
should we raise error/exception here if postproc model is not present?

Reply:
I added that debug log. So, for some cases, we do not want to have postprocess model. I don't think that is an error. It is the expected behavior that we want :)